azure-functions/acbs-function/mappings/facility/helpers/get-maximum-liability.js

```diff
@@ -18,9 +20,7 @@ const getMaximumLiability = (facility, overallAmount = false) => {
     facilityValue = amount;
   }
 
-  const liability = typeof facilityValue === 'string' ? facilityValue.replace(/,/g, '') : facilityValue;
-
-  return Number(Number(liability).toFixed(2));
+  return to2Decimals(facilityValue);
```

Comment:
It doesnt change the implimentation for `undefined`

Before, if it wasn't a string, it would just assign that value (for instance, an `object` or `undefined` or `void` or whatever). It would then call `Number` on it which returns `NaN`

Now, if it's an `object` or `undefined` or `void`, it'll get turned to string (worse calse`'undefined'` for instance) which gets `Number` called on it, which returns `NaN`

Reply:
....unless I've forgotten anything

Reply:
This will return `null` as `NaN` otherwise which would have been `0`, would suggest keeping the existing implementation as agree there is a scope of code refactorisation.

Reply:
I'll update this implimentaiton to conform to this behaviour, although I think in an ideal world we should throw if this is `null` and only let `'0'` or `0` be cooersed into a `0` value as the existing codebase seems to assign `null` a lot